plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -47,27 +47,32 @@ func calculateScore(requested int64, capacity int64, node string) int {
 func calculateResourceOccupancy(pod *api.Pod, node api.Node, nodeInfo *schedulercache.NodeInfo) schedulerapi.HostPriority {
 	totalMilliCPU := nodeInfo.NonZeroRequest().MilliCPU
 	totalMemory := nodeInfo.NonZeroRequest().Memory
+	totalNvidiaGPU := nodeInfo.NonZeroRequest().NvidiaGPU
 	capacityMilliCPU := node.Status.Allocatable.Cpu().MilliValue()
 	capacityMemory := node.Status.Allocatable.Memory().Value()
+	capacityNvidiaGPU := node.Status.Allocatable.NvidiaGPU().Value()
 
 	// Add the resources requested by the current pod being scheduled.
 	// This also helps differentiate between differently sized, but empty, nodes.
 	for _, container := range pod.Spec.Containers {
-		cpu, memory := priorityutil.GetNonzeroRequests(&container.Resources.Requests)
+		cpu, memory, nvidiaGPU := priorityutil.GetNonzeroRequests(&container.Resources.Requests)
 		totalMilliCPU += cpu
 		totalMemory += memory
+		totalNvidiaGPU += nvidiaGPU
 	}
 
 	cpuScore := calculateScore(totalMilliCPU, capacityMilliCPU, node.Name)
 	memoryScore := calculateScore(totalMemory, capacityMemory, node.Name)
+	nvidiaGPUScore := calculateScore(totalNvidiaGPU, capacityNvidiaGPU, node.Name)
 	glog.V(10).Infof(
-		"%v -> %v: Least Requested Priority, Absolute/Requested: (%d, %d) / (%d, %d) Score: (%d, %d)",
+		"%v -> %v: Least Requested Priority, Absolute/Requested: (%d, %d, %d) / (%d, %d, %d) Score: (%d, %d, %d)",
 		pod.Name, node.Name,
-		totalMilliCPU, totalMemory,
-		capacityMilliCPU, capacityMemory,
-		cpuScore, memoryScore,
+		totalMilliCPU, totalMemory, totalNvidiaGPU,
+		capacityMilliCPU, capacityMemory, capacityNvidiaGPU,
+		cpuScore, memoryScore, nvidiaGPUScore,
 	)
 
+	// TODO: what to do with nvidiaGPUScore?
```

Comment:
I converted that and the others into TODOs for me to figure what to do once we have feedback. I would leave them there so anyone reading the code knows we are deliberately doing nothing for now.